release/release.go

```diff
@@ -151,6 +151,38 @@ func CheckUpstreamRelease(ctx context.Context, client *github.Client, org, repo
 	return releases, nil
 }
 
+func KubernetesGoVersion(ctx context.Context, client *github.Client, version string) (string, error) {
+	const (
```

Comment:
This block can be removed. The values are only used once and their usage is clear.